tidy3d/web/api/container.py

```diff
@@ -718,6 +735,10 @@ def pbar_description(task_name: str, status: str) -> str:
                 pbar_tasks = {}
                 for task_name, job in self.jobs.items():
                     status = job.status
+                    if config_end_statuses is not None:
```

Comment:
I'm a little confused, should this logic just go when `end_statuses` is defined at line 712?

```py
end_statuses = config.endStatuses
if end_statuses is None:
    end_statuses = (
        "success",
        "error",
...
```

Reply:
modified